packages/mobile/src/invite/saga.ts

```diff
@@ -275,19 +277,29 @@ function* sendInviteSaga(action: SendInviteAction) {
   }
 }
 
+function* registerAccountDek(newAccount: string) {
```

Comment:
Why create this wrapper?

Reply:
My bad, made an incorrect assumption around the input type of `yield spawn`. Will revert this change and just call `yield spawn` directly with the unwrapped `registerAccountDek`